draft-ietf-add-svcb-dns.md

```diff
@@ -23,7 +23,10 @@ normative:
   RFC2119:
 
 informative:
-
+  FETCH:
+    title: "Fetch Living Standard"
+    date: 11 February 2022
+    target: "https://fetch.spec.whatwg.org/"
```

Comment:
This PR is a worthwhile improvement; I'd like to see it merged. My only comment is that it might be helpful to the reader to link directly to https://fetch.spec.whatwg.org/#port-blocking. Or if that's not a stable URL, to refer to the "Port Blocking" section of https://fetch.spec.whatwg.org/.